kevm-pyk/src/tests/unit/test_foundry_list.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+
+import json
+from functools import cached_property
+from os import listdir
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from kevm_pyk.foundry import foundry_list
+from kevm_pyk.solc_to_k import Contract
+from tests.unit.utils import TEST_DATA_DIR
+
+# from pyk.proof.reachability import APRProof
+
+
+# from unittest.mock import patch
+
+
+if TYPE_CHECKING:
+    from typing import Final
+
+    from pytest_mock import MockerFixture
+
+
+LIST_DATA_DIR: Final = TEST_DATA_DIR / 'foundry-list'
+LIST_OUT: Final = LIST_DATA_DIR / 'out/'
+LIST_APR_PROOF: Final = LIST_OUT / 'apr_proofs/'
+LIST_EXPECTED: Final = LIST_DATA_DIR / 'foundry-list.expected'
+
+
+class FoundryMock:
+    out = LIST_OUT
+
+    @cached_property
+    def contracts(self) -> dict[str, Contract]:
```

Comment:
It would be good to keep mock implementation to the minimum, i.e. have a very simple implementations for `contracts` and `proof_digest`. In order to achieve that try to use simple static values. You can also simplify test data (basically rename files or their content) if necessary.

For example we don't really care if `digest` is realistic, we just want to match the value in the expected output.

Reply:
Thank you so much !!